seacrowd/sea_datasets/culturax/culturax.py

```diff
@@ -0,0 +1,146 @@
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "ssp"
+    SUBSETS = ["id", "jv", "km", "lo", "tl", "min", "my", "su", "th", "vi", "ms", "ceb", "war", "cbk", "bcl"]
+
+    BUILDER_CONFIGS = [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{subset}_source",
+            version=datasets.Version(_SOURCE_VERSION),
+            description=f"{_DATASETNAME} {subset} source schema",
+            schema="source",
+            subset_id=subset,
+        ) for subset in SUBSETS
+    ] + [
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_{subset}_seacrowd_ssp",
+            version=datasets.Version(_SEACROWD_VERSION),
+            description=f"{_DATASETNAME} {subset} SEACrowd schema",
+            schema="seacrowd_ssp",
+            subset_id=subset,
+        )
+        for subset in SUBSETS
+    ]
+
+    DEFAULT_CONFIG_NAME = f"{_DATASETNAME}_jv_source"
+
+    def _info(self) -> datasets.DatasetInfo:
+        if self.config.schema == "source":
+            features = datasets.Features(
+                {
+                    "text": datasets.Value("string"),
+                    "timestamp": datasets.Value("string"),
+                    "url": datasets.Value("string"),
+                    "source": datasets.Value("string"),
+                }
+            )
+        elif self.config.schema == f"seacrowd_{self.SEACROWD_SCHEMA_NAME}":
+            features = schemas.ssp_features
+
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=features,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
+        )
+
+    def _split_generators(self, dl_manager: datasets.DownloadManager) -> List[datasets.SplitGenerator]:
+        """Returns SplitGenerators."""
+        base_path = _BASE_URL.format(lang=self.config.name.split("_")[1])
+
+        checksum_url = urljoin(base_path, "checksum.sha256")
+        checksum_path = Path(dl_manager.download(checksum_url))
+        with open(checksum_path, encoding="utf-8") as f:
+            filenames = [line.split()[1] for line in f if line]
+            data_urls = [urljoin(base_path, filename) for filename in filenames]
+
+        data_paths = list(map(Path, dl_manager.download([url for url in data_urls if url.endswith(".parquet")])))
+
+        return [
+            datasets.SplitGenerator(
+                name=datasets.Split.TRAIN,
+                gen_kwargs={
+                    "filepaths": data_paths,
+                    "split": "train",
+                },
+            )
+        ]
+
+    def _generate_examples(self, filepaths: [Path], split: str) -> Tuple[int, Dict]:
+        """Yields examples as (key, example) tuples.
+
+        Iterate over row groups in each filepaths, then yield each row as an example.
+        https://medium.com/munchy-bytes/are-you-using-parquet-with-pandas-in-the-right-way-595c9ee7112
```

Comment:
I think it's ok to remove this? 

```suggestion
```

Reply:
Oh yeah, I put it there for me to come back because this is my first time working with parquet. It's okay to remove.

Reply:
Ok let's remove it instead :)